### PR TITLE
Config/web.xml

### PR DESCRIPTION
**Pull request by ernestognw**
@livvielin suggested to add an https redirect for every route in the application.

I searched some info about the app.yaml, and it only applies to projects not using the maven app engine plugin, which is deprecated according to [this page.](https://cloud.google.com/appengine/docs/standard/java/using-maven)

Following that spec, the only turnaround I found is to use a web.xml to force https redirection. This is the file added to the project, and it's working correctly in [production](http://ernestognw-step-2020.appspot.com/).

**Note:** First check and approve #6

### CODE REVIEW

File: portfolio/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" version="3.1">
```
Comment (livvielin) [inline portfolio/src/main/webapp/WEB-INF/web.xml:1]: Is there no way this can be https?

Comment (ernestognw) [reply portfolio/src/main/webapp/WEB-INF/web.xml:1]: Seems like I can change it, but it always redirects return the 301 to the following non-https: `http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/index.html`

Even adding https to `https://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/index.html`, the certificate seems not to be valid or something.

I can change it but I think it'll be the same at the end. Any thought?

Comment (livvielin) [reply portfolio/src/main/webapp/WEB-INF/web.xml:1]: I'm mostly extrapolating from https://google.github.io/styleguide/htmlcssguide.html#Protocol even though this is a bit different. I would probably use https where possible even if it doesn't end up making a difference, but if the certificate doesn't work then there's not much we can do and can leave it.

Comment (ernestognw) [reply portfolio/src/main/webapp/WEB-INF/web.xml]: Cool, then I'm only changing it by https, I tried to deploy it and seems not to break anything, so I'm marking this as resolved

**Pull request merged**